railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt

```diff
@@ -54,4 +54,7 @@ Rails.application.configure do
 
   # Raise error when a before_action's only/except options reference missing actions.
   config.action_controller.raise_on_missing_callback_actions = true
+
+  # Disable YJIT for test suite
+  config.yjit = false
```

Comment:
Agreed we should do this, it's an oversight, but we probably should give more context as of why. I'll update your PR.